apps/preferences.py

```diff
@@ -61,6 +61,11 @@ def sync_field(field, dest, default):
             sync_field('label', v, default)
             sync_field('category', v, default)
 
+        if v.get('label'):
+            v['label'] = _(v['label'])
```

Comment:
Gettext will only work with static strings. These translations will be provided when `v['label']` and `v['category']` is assigned a value